drivers/gpio/gpio_stm32.c

```diff
@@ -474,6 +475,15 @@ static int gpio_stm32_config(const struct device *dev,
 		goto exit;
 	}
 
+
+#ifdef CONFIG_PM_DEVICE_RUNTIME
+	/* Enable device clock before configuration (requires bank writes) */
+	err = pm_device_get_sync(dev);
+	if (err < 0 && err != -ENOSYS) {
```

Comment:
when checking for `-ENOSYS` conditional compilation is not needed, since it will never happen.

Reply:
Right. So actually, I'll keep the CONFIG_PM_DEVICE_RUNTIME, as I need to do some more stuff at this point.